php/php.editor/test/unit/data/testfiles/completion/lib/tests225089/tests225089.php.testUseCase2.completion

```diff
@@ -18,5 +18,8 @@ KEYWORD    mixed                                      null
 KEYWORD    null                                       null
 KEYWORD    object                                     null
 KEYWORD    parent                                     null
+KEYWORD    private                                    null
+KEYWORD    protected                                  null
+KEYWORD    public                                     null
```

Comment:
Is this expected? Perhaps this is what @KacerCZ mentioned, right?

Reply:
Yes, it is. The caret(`^`) position is in `__construct()` and there is no parameters.
```php

namespace Froo {
    class Bar {}
}

namespace Trest {
    class TestClass {
        public function __construct(

        /* comment */
^
        ){}

        public function testMethod(

        // comment

        ){}
    }
}

```